src/scripts/install_machine.py

```diff
@@ -46,7 +46,7 @@ def get_graphics_capabilies(connection):
 
 def prepare_graphics_params(connection):
     graphics_config = {
-        'spice': {'listen': '127.0.0.1'},
+        'spice': {'listen': '::1'},
         'vnc': {'listen': '127.0.0.1'}
```

Comment:
Could you add a comment to the bug, so in the future we know why we use ipv6 for spice

```suggestion
        # https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
        'vnc': {'listen': '127.0.0.1'}
```